ruby-challenges.rb

```diff
@@ -19,10 +35,41 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def sum_of_array(array)
+    array.sum
+end
+
+# p sum_of_array(nums_array1)
+# p sum_of_array(nums_array2)
 
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
+class Bike
+   
+    def initialize(model, wheels = 2, current_speed = 0)
+        @model = model
+        @wheels = wheels
+        @current_speed = current_speed
+    end
+    def bike_info
+        p "This #{@model} got #{@wheels} and it's current speed is #{@current_speed}."  
+    end
+    def pedal_faster(increase_speed)
+        @current_speed = @current_speed + increase_speed
+    end
+    def brake(decrease_speed)
+        @current_speed = @current_speed - decrease_speed
+    end
+end
+
+my_bike = Bike.new("Schwin")
+p my_bike.pedal_faster(15)
+p my_bike.pedal_faster(23)
+p my_bike.brake(6)
+p my_bike.brake(13)
+
```

Comment:
Missing the printout the answers the following question
"Create a bike_info method that returns a sentence with all the data from the bike object."